src/main/java/teammates/storage/sqlentity/FeedbackResponse.java

```diff
@@ -43,7 +43,7 @@ public abstract class FeedbackResponse extends BaseEntity {
     @JoinColumn(name = "questionId")
     private FeedbackQuestion feedbackQuestion;
 
-    @OneToMany(mappedBy = "feedbackResponse", cascade = CascadeType.REMOVE)
+    @OneToMany(mappedBy = "feedbackResponse", cascade = {CascadeType.REMOVE, CascadeType.PERSIST})
```

Comment:
New to this, why do you have to add `CascadeType.PERSIST` here?

Reply:
`CascadeType.PERSIST` will make sure that persisting the FeedbackResponse entity will also persist the associated FeedbackResponseComments within the entity.

Reply:
On second thoughts, I think automatically cascading the persist is not very performant for now. 

There are many occasions where the response is updated for other details. Manually cascading will also take advantage of the current `OPTIMIZED_SAVING_POLICY_APPLIED` we have.